__tests__/app.test.js

```diff
@@ -104,4 +104,54 @@ describe('GET api/users', () => {
         expect(msg).toBe('page cannot be found.')
       })
   });
+});
+
+describe.only('PATCH "api/articles/:article_id"', () => {
+  test('status 200, responds with the updated article and correctly amended votes', () => {
+    return request(app)
+    .patch("/api/articles/2")
+    .send({ 'inc_votes' : 100 })
+    .expect(200)
+    .then(({ body }) => {
+      const { article } = body
+      expect(article).toEqual(
+        {
+          article_id: 2,
+          title: "Sony Vaio; or, The Laptop",
+          topic: "mitch",
+          author: "icellusedkars",
+          body: "Call me Mitchell. Some years ago—never mind how long precisely—having little or no money in my purse, and nothing particular to interest me on shore, I thought I would buy a laptop about a little and see the codey part of the world. It is a way I have of driving off the spleen and regulating the circulation. Whenever I find myself growing grim about the mouth; whenever it is a damp, drizzly November in my soul; whenever I find myself involuntarily pausing before coffin warehouses, and bringing up the rear of every funeral I meet; and especially whenever my hypos get such an upper hand of me, that it requires a strong moral principle to prevent me from deliberately stepping into the street, and methodically knocking people’s hats off—then, I account it high time to get to coding as soon as I can. This is my substitute for pistol and ball. With a philosophical flourish Cato throws himself upon his sword; I quietly take to the laptop. There is nothing surprising in this. If they but knew it, almost all men in their degree, some time or other, cherish very nearly the same feelings towards the the Vaio with me.",
+          created_at: '2020-10-16T05:03:00.000Z',
+          votes: 100,
+        })
+    })
+  });
+  test('can update the votes with a negative number', () => {
+    return request(app)
+    .patch("/api/articles/4")
+    .send({ 'inc_votes' : -100 })
+    .expect(200)
+    .then(({ body }) => {
+      const { article } = body
+      expect(article.votes).toBe(-100)
+    })
+  });
+  test('status 404, responds with not found when wrong article id put in', () => {
+    return request(app)
+    .patch("/api/articles/20000")
+    .send({ 'inc_votes' : 100 })
+    .expect(404)
+    .then(({ body }) => {
+      expect(body.msg).toBe('no article found with that id')
+    })
+  });
+  test('status 400, when the article_id is not a number', () => {
+    return request(app)
+    .patch("/api/articles/SELECT * FROM articles")
+    .send({ 'inc_votes' : 100 })
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe('Bad request');
+    })
+  });
 });
```

Comment:
I would maybe think about error handling for missing / incorrect body, e.g. `inc_votes` property is not a number, or missing? How would we handle this?